### ISSUE TITLE
aws_dx_private_virtual_interface: 8500 MTU

### ISSUE THREAD

**Issue by Midacts** [terraform-providers/terraform-provider-aws#8841]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

```Terraform v0.11.10```
```provider.aws v2.12.0```

### Affected Resource(s)

* aws_dx_private_virtual_interface

### Terraform Configuration Files

```hcl
resource "aws_dx_private_virtual_interface" "ex" {
  connection_id   = "${aws_dx_connection.ex.id}"
  name            = "ex"
  vlan            = 100
  address_family  = "ipv4"
  bgp_asn         = 64xxx
  mtu             = 8500
  dx_gateway_id   = "xxx"
}
```

### Debug Output

Error: aws_dx_private_virtual_interface.ex: expected mtu to be one of [1500 9001], got 8500

### Expected Behavior

Create/import the dx vif with jumbo frames (8500 MTU)

### Actual Behavior

Errors out

### Important Factoids

![image](https://user-images.githubusercontent.com/3960208/58812734-713d5180-85f0-11e9-8b92-25f0fbc570e7.png)

**Comment by ewbankkit** [comment]
Hi @Midacts, Is this a private virtual interface or transit virtual interface that you're trying to import?
The [AWS Direct Connect docs](https://docs.aws.amazon.com/directconnect/latest/UserGuide/set-jumbo-frames-vif.html) are a bit hard to parse exactly but my understanding was that an MTU of 8500 was only for transit VIFs, where we are supporting 8500 MTUs in https://github.com/terraform-providers/terraform-provider-aws/pull/8522.

**Comment by Midacts** [comment]
@ewbankkit Thanks for the response. I think you are right.
Thanks for the link to the open issue. It makes sense that that is the option (I didn't know it was an option because it technically does not exist).

Want me to close this issue? I should be able to watch #8522

**Comment by ewbankkit** [comment]
@Midacts Thanks. Please 👍 #8522 if you didn't already.
We ought to add code to ensure that the `terraform import` is actually importing the correct type of VIF (private/public/transit) - I'll open an issue for that and link to this one.
Yes, please go ahead and close this issue.
